data/fleet-commander-standalone.in

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [ $UID != 0 ]; then
+    echo "WARNING: You need to be root in order to have access to dbus service"
+fi
+
+PYTHONPATH=@FCPYTHONDIR@ @PYTHON@ -m fleetcommander.admin $*
```

Comment:
Are we going to run this as root? I thought we agreed this would run with su -c ... - fleet-commander-admin